stage1/dbus/units/org.freedesktop.machine1.busname

```diff
@@ -0,0 +1,15 @@
+#  This file is part of systemd.
```

Comment:
What is this file for? Doesn't systemd provide it? Also, it's about machined, which is disabled in stage1.